esp/src/lws.config.js

```diff
@@ -1,4 +1,4 @@
-const debugServerIP = "192.168.99.103";
+const debugServerIP = "10.176.152.166";
```

Comment:
Doesn't deel like information that should be checked in (even if it's an internal IP)

Reply:
Agreed will remove and fix up the commit message as well.